Objects/listobject.c

```diff
@@ -2145,12 +2145,21 @@ list.sort
     key as keyfunc: object = None
     reverse: bool(accept={int}) = False
 
-Stable sort *IN PLACE*.
+Sort the list in ascending order.
+
+The sort is in-place (i.e. the list itself is modified) and stable (i.e. the
+order of two equal elements is maintained).
+
+A key function can be supplied to customize the sorting behavior. The key
+function is applied to each list element, and the elements are sorted
+according to the values returned by the key function.
+
+The reverse flag can be set to sort in descending order.
```

Comment:
Change to the following:
"If *reverse* is True, sort in in descending order instead."
Move this right after the initial line to make immediately clear that 'ascending order' is just the default.

Reply:
I prefer the OP's version which reads like plain English.

Reply:
I changed my mind about moving the sentence.  I still like my reverse better, but this overall is a great improvement.